commit 9570a7f6af152e2f1e63181ad8a20d12a10e3c5b
Author: Tom Tromey <tromey@redhat.com>
Date:   Mon Aug 9 15:22:51 2010 +0300

    Create GDB index from find-debuginfo if possible (RhBug:617166)
    - Details in https://fedoraproject.org/wiki/Features/GdbIndex

diff --git a/scripts/find-debuginfo.sh b/scripts/find-debuginfo.sh
index 01456da..3ada962 100644
--- a/scripts/find-debuginfo.sh
+++ b/scripts/find-debuginfo.sh
@@ -207,6 +207,8 @@ while read nlinks inum f; do
     $strict && exit 2
   fi
 
+  [ -x /usr/bin/gdb-add-index ] && /usr/bin/gdb-add-index "$f" > /dev/null 2>&1
+
   # A binary already copied into /usr/lib/debug doesn't get stripped,
   # just has its file names collected and adjusted.
   case "$dn" in
